pcdsdevices/sxr_test_absorber.py

```diff
@@ -0,0 +1,22 @@
+"""
+Module for the SXR Test Absorbers.
+"""
+from ophyd.device import Component as Cpt
+from ophyd.device import Device
+
+from .epics_motor import BeckhoffAxis
+
+
+class SxrTestAbsorber(Device):
+    """
+    SXR Test Absorber: Used for testing the sxr beamline at high pulse rates.
+
+    This device has 1 main members: the stopper/absorber (diamond stopper).
+
+    The vertical motor moves stopper in and out of the beam path in
+    the +/- y direction.
+    """
+
+    tab_component_names = True
+
+    absorber_vert = Cpt(BeckhoffAxis, ':absorber_vert', kind='normal')
```

Comment:
```suggestion
    absorber_vert = Cpt(BeckhoffAxis, ':MMS', kind='normal')
```
Since the PV here is `{attribute 'pytmc' := 'pv: ST1K4:TEST:MMS'}`
And I assume the base PV of the device is `ST1K4:TEST`